index.html

```diff
@@ -10323,9 +10327,9 @@ <h3>Operating System Accessibility API mapping of multi-value ARIA attributes</h
 			<h3>ARIA nullable DOMString Attributes</h3>
 			<p>As noted in [[[#typemapping]]], attributes are included in host languages, and the syntax for representation of WAI-ARIA types is governed by the host language.</p>
 			<p>The following algorithm should be used for ARIA nullable {{DOMString}} attributes in HTML:</p>
-			<p>On getting, if the corresponding content attribute is not present, then the IDL attribute must return null,
+			<p>On getting, if the corresponding content attribute is not present, then the IDL attribute must return <code>null</code>,
 			otherwise, the IDL attribute must get the value in a transparent, case-preserving manner.
-			On setting, if the new value is null, the content attribute must be removed, and otherwise,
+			On setting, if the new value is nullish (<code>null</code>code> or <code>undefined</code>), the content attribute must be removed, and otherwise,
```

Comment:
This is handled by IDL, you wouldn't get undefined here.

Reply:
Is there a better way to phrase this?

Reply:
Yes, not saying anything as this already follows from https://webidl.spec.whatwg.org/#es-nullable-type.

Reply:
So to be clear, do you recommend removing both of these paragraphs entirely?

> The following algorithm should […]
> 
> On getting, […]

Reply:
No, this PR isn't needed, that's all.

Reply:
In that case, I'm happy to close this PR! Thanks.